2015/09/25 15:59:25 [warning] [application] Не удалось присвоить небезопасный атрибут "username" класса "LoginForm".
in /var/www/madali.am/protected/controllers/SiteController.php (92)
in /var/www/madali.am/index.php (13)
2015/09/25 15:59:25 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.Հեռախոսահամար".' in /var/www/madali.am/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali.am/framework/validators/CRequiredValidator.php(51): CComponent->__get('???????????????...')
#1 /var/www/madali.am/framework/validators/CValidator.php(213): CRequiredValidator->validateAttribute(Object(LoginForm), '???????????????...')
#2 /var/www/madali.am/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 /var/www/madali.am/protected/controllers/SiteController.php(94): CModel->validate()
#4 /var/www/madali.am/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#5 /var/www/madali.am/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/madali.am/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/madali.am/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/madali.am/framework/web/CWebApplication.php(282): CController->run('login')
#9 /var/www/madali.am/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#10 /var/www/madali.am/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /var/www/madali.am/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/madali.am/index.php?r=site/login
HTTP_REFERER=http://localhost/madali.am/index.php?r=site/login
---
2015/09/25 16:11:07 [warning] [application] Не удалось присвоить небезопасный атрибут "username" класса "LoginForm".
in /var/www/madali.am/protected/controllers/SiteController.php (92)
in /var/www/madali.am/index.php (13)
2015/09/25 16:11:07 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.user".' in /var/www/madali.am/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali.am/framework/validators/CRequiredValidator.php(51): CComponent->__get('user')
#1 /var/www/madali.am/framework/validators/CValidator.php(213): CRequiredValidator->validateAttribute(Object(LoginForm), 'user')
#2 /var/www/madali.am/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 /var/www/madali.am/protected/controllers/SiteController.php(94): CModel->validate()
#4 /var/www/madali.am/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#5 /var/www/madali.am/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/madali.am/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/madali.am/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/madali.am/framework/web/CWebApplication.php(282): CController->run('login')
#9 /var/www/madali.am/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#10 /var/www/madali.am/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /var/www/madali.am/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/madali.am/index.php?r=site/login
HTTP_REFERER=http://localhost/madali.am/index.php?r=site/login
---
2015/09/26 10:43:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "users".' in /var/www/madali/framework/web/CController.php:483
Stack trace:
#0 /var/www/madali/framework/web/CController.php(270): CController->missingAction('users')
#1 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('users')
#2 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/users')
#3 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/madali/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/madali/index.php?r=site/users
HTTP_REFERER=http://localhost/madali/index.php?r=site/index
---
2015/09/26 10:44:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "../users".' in /var/www/madali/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('../users')
#1 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/madali/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/madali/index.php?r=../users
HTTP_REFERER=http://localhost/madali/index.php?r=site/index
---
2015/09/26 10:45:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "views/users".' in /var/www/madali/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('views/users')
#1 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/madali/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/madali/index.php?r=views/users
HTTP_REFERER=http://localhost/madali/index.php?r=site/index
---
2015/09/26 10:47:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "users".' in /var/www/madali/framework/web/CController.php:483
Stack trace:
#0 /var/www/madali/framework/web/CController.php(270): CController->missingAction('users')
#1 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('users')
#2 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/users')
#3 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/madali/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/madali/index.php?r=site/users
HTTP_REFERER=http://localhost/madali/index.php?r=site/index
---
2015/09/26 14:50:39 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/web/helpers/CHtml.php(2255): CActiveRecord->__get('username')
#2 /var/www/madali/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Users), 'username')
#3 /var/www/madali/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Users), 'username', Array)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(Users), 'username', Array)
#5 /var/www/madali/protected/views/users/index.php(25): CActiveForm->textField(Object(Users), 'username')
#6 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#7 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#8 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#10 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#11 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#12 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#16 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#17 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /var/www/madali/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/
---
2015/09/26 14:54:37 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.verifyCode".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('verifyCode')
#1 /var/www/madali/framework/web/helpers/CHtml.php(2255): CActiveRecord->__get('verifyCode')
#2 /var/www/madali/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Users), 'verifyCode')
#3 /var/www/madali/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Users), 'verifyCode', Array)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(Users), 'verifyCode', Array)
#5 /var/www/madali/protected/views/users/index.php(60): CActiveForm->textField(Object(Users), 'verifyCode')
#6 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#7 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#8 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#10 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#11 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#12 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#16 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#17 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /var/www/madali/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/
---
2015/09/26 14:56:19 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.verifyCode".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('verifyCode')
#1 /var/www/madali/framework/web/helpers/CHtml.php(2255): CActiveRecord->__get('verifyCode')
#2 /var/www/madali/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Users), 'verifyCode')
#3 /var/www/madali/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Users), 'verifyCode', Array)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(Users), 'verifyCode', Array)
#5 /var/www/madali/protected/views/users/index.php(60): CActiveForm->textField(Object(Users), 'verifyCode')
#6 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#7 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#8 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#10 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#11 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#12 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#16 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#17 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /var/www/madali/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/
---
2015/09/26 14:58:05 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.verifyCode".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('verifyCode')
#1 /var/www/madali/framework/web/helpers/CHtml.php(2255): CActiveRecord->__get('verifyCode')
#2 /var/www/madali/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Users), 'verifyCode')
#3 /var/www/madali/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Users), 'verifyCode', Array)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(Users), 'verifyCode', Array)
#5 /var/www/madali/protected/views/users/index.php(60): CActiveForm->textField(Object(Users), 'verifyCode')
#6 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#7 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#8 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#10 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#11 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#12 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#16 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#17 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /var/www/madali/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/
---
2015/09/26 15:00:16 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:16 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:16 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:16 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:00:24 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:24 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:24 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:24 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:00:27 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:27 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:27 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:27 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:00:46 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:46 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:46 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:46 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:00:51 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:51 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:51 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:00:51 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:00:53 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (32)
in /var/www/madali/index.php (13)
2015/09/26 15:00:53 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (32)
in /var/www/madali/index.php (13)
2015/09/26 15:00:53 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (32)
in /var/www/madali/index.php (13)
2015/09/26 15:00:53 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/protected/controllers/UsersController.php(33): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:01 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:01 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:01 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:01 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:09 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:09 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:09 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:09 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:11 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:11 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:11 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:11 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:12 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:12 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:12 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:12 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:18 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:18 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:18 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:18 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:20 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:20 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:20 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:20 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:33 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:33 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:33 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:33 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:38 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:38 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:38 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (26)
in /var/www/madali/index.php (13)
2015/09/26 15:01:38 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#5 /var/www/madali/protected/controllers/UsersController.php(26): CActiveForm::validate(Object(Users))
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:01:38 [warning] [application] Не удалось присвоить небезопасный атрибут "name" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (32)
in /var/www/madali/index.php (13)
2015/09/26 15:01:38 [warning] [application] Не удалось присвоить небезопасный атрибут "surname" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (32)
in /var/www/madali/index.php (13)
2015/09/26 15:01:38 [warning] [application] Не удалось присвоить небезопасный атрибут "phone" класса "Users".
in /var/www/madali/protected/controllers/UsersController.php (32)
in /var/www/madali/index.php (13)
2015/09/26 15:01:38 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.username".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('username')
#1 /var/www/madali/framework/validators/CUniqueValidator.php(81): CActiveRecord->__get('username')
#2 /var/www/madali/framework/validators/CValidator.php(213): CUniqueValidator->validateAttribute(Object(Users), 'username')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(Users), NULL)
#4 /var/www/madali/protected/controllers/UsersController.php(33): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:04:02 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.pwd_hash".' in /var/www/madali/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(159): CComponent->__set('pwd_hash', '$6$N6.BAfaz$m8l...')
#1 /var/www/madali/protected/models/Users.php(105): CActiveRecord->__set('pwd_hash', '$6$N6.BAfaz$m8l...')
#2 /var/www/madali/framework/db/ar/CActiveRecord.php(1017): Users->beforeSave()
#3 /var/www/madali/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#4 /var/www/madali/protected/controllers/UsersController.php(35): CActiveRecord->save()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:13:19 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(25): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:13:22 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(25): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:14:12 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(25): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:14:21 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(25): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:16:19 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(20): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:16:35 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(20): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:18:14 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:18:18 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:20:19 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:21:56 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:21:59 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:22:10 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:23:27 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/models/LoginForm.php(53): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#2 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#3 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#4 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#5 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#6 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#10 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#11 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#12 /var/www/madali/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:24:40 [warning] [application] Не удалось присвоить небезопасный атрибут "username" класса "LoginForm".
in /var/www/madali/protected/controllers/SiteController.php (92)
in /var/www/madali/index.php (13)
2015/09/26 15:24:40 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/validators/CRequiredValidator.php(51): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CValidator.php(213): CRequiredValidator->validateAttribute(Object(LoginForm), 'phone')
#2 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#4 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#5 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#9 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#10 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /var/www/madali/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:25:12 [warning] [application] Не удалось присвоить небезопасный атрибут "username" класса "LoginForm".
in /var/www/madali/protected/controllers/SiteController.php (92)
in /var/www/madali/index.php (13)
2015/09/26 15:25:12 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "LoginForm.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/validators/CRequiredValidator.php(51): CComponent->__get('phone')
#1 /var/www/madali/framework/validators/CValidator.php(213): CRequiredValidator->validateAttribute(Object(LoginForm), 'phone')
#2 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#4 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#5 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#9 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#10 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /var/www/madali/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:26:23 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "UserIdentity.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/protected/components/UserIdentity.php(20): CComponent->__get('phone')
#1 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#2 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/madali/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:26:50 [error] [exception.CDbException] exception 'CDbException' with message 'Таблица "users" не содержит столбца "username".' in /var/www/madali/framework/db/schema/CDbCommandBuilder.php:591
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(1464): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 /var/www/madali/protected/components/UserIdentity.php(20): CActiveRecord->findByAttributes(Array)
#2 /var/www/madali/protected/models/LoginForm.php(54): UserIdentity->authenticate()
#3 /var/www/madali/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 /var/www/madali/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 /var/www/madali/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 /var/www/madali/protected/controllers/SiteController.php(94): CModel->validate()
#7 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#8 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#10 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#12 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#13 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#14 /var/www/madali/index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:27:52 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('phone')
#1 /var/www/madali/framework/web/helpers/CHtml.php(2255): CActiveRecord->__get('phone')
#2 /var/www/madali/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Users), 'phone')
#3 /var/www/madali/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Users), 'phone', Array)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(Users), 'phone', Array)
#5 /var/www/madali/protected/views/users/index.php(51): CActiveForm->textField(Object(Users), 'phone')
#6 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#7 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#8 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#10 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#11 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#12 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#16 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#17 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /var/www/madali/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=site/index
---
2015/09/26 15:28:39 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Users.phone".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/db/ar/CActiveRecord.php(143): CComponent->__get('phone')
#1 /var/www/madali/framework/web/helpers/CHtml.php(2255): CActiveRecord->__get('phone')
#2 /var/www/madali/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Users), 'phone')
#3 /var/www/madali/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Users), 'phone', Array)
#4 /var/www/madali/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(Users), 'phone', Array)
#5 /var/www/madali/protected/views/users/index.php(51): CActiveForm->textField(Object(Users), 'phone')
#6 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#7 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#8 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#10 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#11 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#12 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#16 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#17 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /var/www/madali/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php?r=site/index
---
2015/09/26 15:37:26 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CWebUser.surname".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/web/auth/CWebUser.php(140): CComponent->__get('surname')
#1 /var/www/madali/protected/views/layouts/main.php(49): CWebUser->__get('surname')
#2 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#3 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#4 /var/www/madali/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/madali...', Array, true)
#5 /var/www/madali/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 /var/www/madali/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 /var/www/madali/framework/web/CBaseController.php(206): COutputProcessor->run()
#8 /var/www/madali/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 /var/www/madali/protected/views/layouts/column1.php(6): CBaseController->endContent()
#10 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#11 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#12 /var/www/madali/framework/web/CController.php(784): CBaseController->renderFile('/var/www/madali...', Array, true)
#13 /var/www/madali/protected/controllers/SiteController.php(32): CController->render('index')
#14 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#15 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#16 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#17 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#19 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#20 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /var/www/madali/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/madali/index.php
HTTP_REFERER=http://localhost/madali/index.php?r=site/login
---
2015/09/26 15:44:50 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CWebUser.surname".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/web/auth/CWebUser.php(140): CComponent->__get('surname')
#1 /var/www/madali/protected/views/layouts/main.php(49): CWebUser->__get('surname')
#2 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#3 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#4 /var/www/madali/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/madali...', Array, true)
#5 /var/www/madali/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 /var/www/madali/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 /var/www/madali/framework/web/CBaseController.php(206): COutputProcessor->run()
#8 /var/www/madali/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 /var/www/madali/protected/views/layouts/column1.php(6): CBaseController->endContent()
#10 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#11 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#12 /var/www/madali/framework/web/CController.php(784): CBaseController->renderFile('/var/www/madali...', Array, true)
#13 /var/www/madali/protected/controllers/SiteController.php(98): CController->render('login', Array)
#14 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#15 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#16 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#17 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('login')
#19 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#20 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /var/www/madali/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/madali/index.php?r=site/login
HTTP_REFERER=http://localhost/madali/index.php?r=users
---
2015/09/26 15:44:56 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CWebUser.surname".' in /var/www/madali/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/madali/framework/web/auth/CWebUser.php(140): CComponent->__get('surname')
#1 /var/www/madali/protected/views/layouts/main.php(49): CWebUser->__get('surname')
#2 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#3 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#4 /var/www/madali/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/madali...', Array, true)
#5 /var/www/madali/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 /var/www/madali/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 /var/www/madali/framework/web/CBaseController.php(206): COutputProcessor->run()
#8 /var/www/madali/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 /var/www/madali/protected/views/layouts/column1.php(6): CBaseController->endContent()
#10 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#11 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#12 /var/www/madali/framework/web/CController.php(784): CBaseController->renderFile('/var/www/madali...', Array, true)
#13 /var/www/madali/protected/controllers/SiteController.php(32): CController->render('index')
#14 /var/www/madali/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#15 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#16 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#17 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#19 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#20 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /var/www/madali/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/madali/
---
2015/09/26 16:05:09 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CUniqueValidator.0".' in /var/www/madali/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/madali/framework/validators/CValidator.php(189): CComponent->__set('0', 'required')
#1 /var/www/madali/framework/base/CModel.php(286): CValidator::createValidator('unique', Object(Users), 'name', Array)
#2 /var/www/madali/framework/base/CModel.php(260): CModel->createValidators()
#3 /var/www/madali/framework/base/CModel.php(303): CModel->getValidators('name')
#4 /var/www/madali/framework/web/helpers/CHtml.php(1237): CModel->isAttributeRequired('name')
#5 /var/www/madali/framework/web/widgets/CActiveForm.php(562): CHtml::activeLabelEx(Object(Users), 'name', Array)
#6 /var/www/madali/protected/views/users/index.php(24): CActiveForm->labelEx(Object(Users), 'name')
#7 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#8 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#10 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#11 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#12 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#13 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#17 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#18 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /var/www/madali/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php
---
2015/09/26 16:05:55 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CStringValidator.0".' in /var/www/madali/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/madali/framework/validators/CValidator.php(189): CComponent->__set('0', 'required')
#1 /var/www/madali/framework/base/CModel.php(286): CValidator::createValidator('length', Object(Users), 'surname, gender', Array)
#2 /var/www/madali/framework/base/CModel.php(260): CModel->createValidators()
#3 /var/www/madali/framework/base/CModel.php(303): CModel->getValidators('name')
#4 /var/www/madali/framework/web/helpers/CHtml.php(1237): CModel->isAttributeRequired('name')
#5 /var/www/madali/framework/web/widgets/CActiveForm.php(562): CHtml::activeLabelEx(Object(Users), 'name', Array)
#6 /var/www/madali/protected/views/users/index.php(24): CActiveForm->labelEx(Object(Users), 'name')
#7 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#8 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#10 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#11 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#12 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#13 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#17 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#18 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /var/www/madali/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php
---
2015/09/26 16:08:25 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CEmailValidator.0".' in /var/www/madali/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/madali/framework/validators/CValidator.php(189): CComponent->__set('0', 'required')
#1 /var/www/madali/framework/base/CModel.php(286): CValidator::createValidator('email', Object(Users), 'email', Array)
#2 /var/www/madali/framework/base/CModel.php(260): CModel->createValidators()
#3 /var/www/madali/framework/base/CModel.php(303): CModel->getValidators('name')
#4 /var/www/madali/framework/web/helpers/CHtml.php(1237): CModel->isAttributeRequired('name')
#5 /var/www/madali/framework/web/widgets/CActiveForm.php(562): CHtml::activeLabelEx(Object(Users), 'name', Array)
#6 /var/www/madali/protected/views/users/index.php(24): CActiveForm->labelEx(Object(Users), 'name')
#7 /var/www/madali/framework/web/CBaseController.php(126): require('/var/www/madali...')
#8 /var/www/madali/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/madali...', Array, true)
#9 /var/www/madali/framework/web/CController.php(869): CBaseController->renderFile('/var/www/madali...', Array, true)
#10 /var/www/madali/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#11 /var/www/madali/protected/controllers/UsersController.php(42): CController->render('index', Array)
#12 /var/www/madali/framework/web/actions/CInlineAction.php(49): UsersController->actionIndex()
#13 /var/www/madali/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /var/www/madali/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /var/www/madali/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /var/www/madali/framework/web/CWebApplication.php(282): CController->run('')
#17 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#18 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /var/www/madali/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/madali/index.php?r=users
HTTP_REFERER=http://localhost/madali/index.php
---
2015/09/26 17:21:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "booking".' in /var/www/madali/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/madali/framework/web/CWebApplication.php(141): CWebApplication->runController('booking')
#1 /var/www/madali/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/madali/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/madali/index.php?r=booking
HTTP_REFERER=http://localhost/madali/index.php?r=profile
---
